src/newrelic/app.go

```diff
@@ -21,7 +21,7 @@ import (
 	"newrelic/utilization"
 )
 
-// AgentRunID is a string as of agent listener protocol version 14.
```

Comment:
I think the comment was fine as is as it is saying AgentRunID has been a string since v14?

Reply:
Good catch